dockstore-client/src/main/java/io/dockstore/client/cli/nested/AbstractEntryClient.java

```diff
@@ -531,22 +531,39 @@ private void handleEntry2tsv(List<String> args) throws ApiException, IOException
      * @return
      */
     public Boolean checkCWL(File content){
```

Comment:
Ok, getting better. 
I think this just lacks documentation (fill out the Javadoc to explain what your method does) and then try to explain what you're trying to calculate at each major decision point in these two methods. Thanks!